app/models/spam_domain.rb

```diff
@@ -4,6 +4,7 @@ class SpamDomain < ApplicationRecord
   include Websocket
 
   has_and_belongs_to_many :posts, after_add: :setup_review
```

Comment:
Likewise, `has_many :through`

Reply:
Can do, the only reason I didn't the first time is because I didn't want rails to be angry at me for changing things. It was already angry about the composite primary key